spec/features/users/facebook_login_spec.rb

```diff
@@ -39,6 +39,15 @@
 
         expect(current_path).to eq("/profile")
         expect(page).to have_content("You are now logged in as Sharon")
+
```

Comment:
I am changing some stuff at the beginning of this test, but it should be fine to merge because this is lower down.